### DISCUSSION TITLE
How to create next word prediction?

### DISCUSSION THREAD

**Discussion by yudhafebrian** [AnySoftKeyboard/AnySoftKeyboard#3256] [Q&A]
Hello, anybody know how to create next word prediction? current keyboard only give prediction of what we half typing, but not the next word/sentences just like in Gboard or Swiftkey

here is the example fro Swiftkey:
![image](https://user-images.githubusercontent.com/36806393/154821577-0b436660-8953-407d-b0b0-1c71b80408b2.png)

**Comment by sslaia** [comment]
I've very limited knowledge about the next word prediction. However what I can see in other language addons and I applied to my language pack, for this you need the word frequency number between 0 - 255 (you can see the existing dictionary files).
How to count word frequency? You can scrap the web or use other text files and count the frequency (using python for example). Then convert the frequency numbers to 0-255.

**Comment by AT85-UG** [comment]
Hi,  i noticed missing next word prediction in my AnySoftKeyboard. Did you find a solution, how to activate "next word prediction"? Would be interesting to get to know how. Thx Marcus
